smartmeter/smartmeter.ino

```diff
@@ -22,6 +22,8 @@
 #define YELLOW_LED D7
 #define GREEN_LED D4
 #define RELAY D2
+#define CURRENT_SENSOR A0
+#define BUTTON 14 // D5
```

Comment:
Why are you using 14 and not D5 ?

Reply:
It was a quick test. I knew that the correct pin was 14, but not sure if it was indeed routed to D5, but it was and I forgot to change. Changing

Reply:
Done